src/common/transformations/include/ov_ops/vl_sdpa.hpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+#include "openvino/op/util/sub_graph_base.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace op {
+namespace internal {
+class TRANSFORMATIONS_API VLSDPA : public ov::op::Op {
```

Comment:
Please add description for the operation. Specifically, it would be helpful to include how it differs from SDPA, its use cases, and description of the expected inputs and shapes